JSDemos/Demos/Slider/Overview/Angular/description.md

```diff
@@ -0,0 +1,45 @@
+This demo shows how you can create and configure a Slider.
+
+## Create a Slider
+
+To create a Slider, declare it in markup and use the [min](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#min) and [max](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#max) properties to limit the range of accepted values.
+
+You can also specify the [value](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#value) property to change the initial value.
+
+The **Default mode** section shows a Slider with such a basic setup.
+
+## Customize Slider Appearance
+
+### Max and min labels
+
+The Slider can display labels for the [min](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#min) and [max](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#max) values. To configure the labels, use the [label](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/label/) object. In this object, specify the [visible](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/label/#visible), [position](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/label/#position), and [format](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/label/#format) properties.
+
+### Handle tooltip
```

Comment:
Now that I read this, it seems a bit confusing - I first assume that "handle" is a verb. So maybe change this caption to simply "Tooltips"? 

```suggestion
### Tooltips
```